actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -71,6 +72,7 @@
   private static InterServiceCommunication interServiceCommunication =
       InterServiceCommunicationFactory.getInstance();
   private ActorRef systemSettingActorRef = null;
+  private static final String PUBLIC_ROLE = "PUBLIC";
```

Comment:
User ProjectUtils. UserRole instead of private static final String PUBLIC_ROLE = "PUBLIC";